database only restore API flag (#3573)
mendix/docs: content/apidocs-mxsdk/apidocs/backups-api.md

**Query Parameters**

*   _source\_snapshot\_id_ (String): Identifier of the snapshot which will be restored. This value is required and must belong to a snapshot within the same application, although it could be a different environment.

> *   _db\_only_ (Boolean): Boolean flag. Set this to *true* if you are doing a database only restore operation. It defaults to *false* if not present.

Reviewer: @joshp123 
Hi Josh, how about adding the following big orange warning?
```suggestion
*   _db\_only_ (Boolean): Boolean flag. Set this to *true* if you are doing a database only restore operation. It defaults to *false* if not present.

    {{% alert type="warning" %}}Setting this to `true` will not restore any of your files and so there is a risk that data will be missing from your app or that your app will not work as expected. Use this option with caution.{{% /alert %}}
```

Reviewer 2: Thanks @MarkvanMents , that looks great!

> *   _db\_only_ (Boolean): Boolean flag. It might have `true` or `false` value. It determines if it is a database only restore operation. It defaults to false if not present.

Reviewer: I think it might be better to say what the true value means. Perhaps
```suggestion
*   _db\_only_ (Boolean): Boolean flag. Set this to *true* if you are doing a database only restore operation. It defaults to *false* if not present.
```
What do you think?
